packages/realm/src/app-services/Credentials.ts

```diff
@@ -60,13 +60,21 @@ export class Credentials {
 
   /**
    * Creates credentials based on a login with an email address and a password.
-   * @param credentials An object with username and password for the user.
    * @return {Credentials} An instance of `Credentials` that can be used in {@linkcode App.logIn}.
    */
-  static emailPassword(credentials: { email: string; password: string }): Credentials {
-    assert.string(credentials.email, "email");
-    assert.string(credentials.password, "password");
-    return new Credentials(binding.AppCredentials.usernamePassword(credentials.email, credentials.password));
+  static emailPassword(credentials: { email: string; password: string }): Credentials;
+  static emailPassword(email: string, password: string): Credentials;
+  static emailPassword(arg1: { email: string; password: string } | string, password?: string): Credentials {
+    let email;
+    if (typeof arg1 === "string") {
+      email = arg1;
```

Comment:
Either do `const email = arg1` in this scope or just use `arg1` in place of  `email` in the call on line 72. Either way, delete line 68.